.circleci/config.yml

```diff
@@ -100,12 +115,17 @@ jobs:
 
       - restore_cache:
           keys:
-            - -{{ checksum "Makefile.envs" }}-v20201205-
+            - -pkg-{{ checksum "Makefile.envs" }}-v20210911-
 
       - run:
           name: build packages
           no_output_timeout: 1800
           command: |
+            source pyodide_env.sh
+
+            # Set mtime for EM_CONFIG to avoid ccache cache misses
+            touch -m -d '1 Jan 2021 12:00' emsdk/emsdk/.emscripten
```

Comment:
Source: https://github.com/juj/ccache/blob/d69e1d05d68b06a256f00a86a38de644ff23b66d/src/emcc.cpp#L627

This can probably be switched to hashing the file, but mtime+size is the default ccache strategy for compilers apparently (see `CCACHE_COMPILERCHECK`)